cloud/libvirt/actuators/machine/actuator.go

```diff
@@ -62,11 +105,139 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster, machine *clusterv1.Machine
 // Update updates a machine and is invoked by the Machine Controller
 func (a *Actuator) Update(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	glog.Infof("Updating machine %v for cluster %v.", machine.Name, cluster.Name)
-	return fmt.Errorf("TODO: Not yet implemented")
+	error := errorWrapper{cluster: cluster, machine: machine}
```

Comment:
Maybe `errorWr`? I got confused by the `error.WithLog`. The `error` is allocated for `error` type. We should not override it unless necessary.

Reply:
Good point. Fixed.